test/integration/connection-monitoring-and-pooling/connection.test.ts

```diff
@@ -144,6 +144,21 @@ describe('Connection', function () {
         }
       }
     });
+
```

Comment:
Moved this out of unit tests so it could be configured properly / not default to `OpQuery` incorrectly